CIP-Draft-Initial-Params/Initial.md

```diff
@@ -0,0 +1,172 @@
+## Preamble
+
+---
+* CIP: ?
+* Title: Cardano Protocol Parameters
+* Authors: Kevin Hammond (<kevin.hammond@iohk.io>)
+* Discussions-To: <kevin.hammond@iohk.io>
+* Comments-Summary: <summary tone>
+* Comments-URI: <links to wiki page for comments>
+* Status: Draft
+* Type: Standards Track
+* Created: 2020-10-14
+* License: <abbreviation for approved license(s)>
+* License-Code: <abbreviation for code under different approved license(s)>
+* Post-History: <dates of postings to Cardano Dev Forum, or link to thread>
+* Requires: <CIP number(s)>
+* Replaces: <CIP number>
+* Superseded-By: <CIP number>
+---
+
+## Simple Summary/Abstract
+
+This CIP describes the initial protocol parameter settings for the Shelley era of the Cardano blockchain.
+
+## Motivation
+
+We need to provide a concise description of the initial protocol parameter choices, that can be used as the base for future proposed protocol changes.
+
+## Specification
+
+The initial protocol parameters are given below (in JSON format):
+
+```
+{
+    "protocolVersion": {
+        "major": 2,
+        "minor": 0
+    },
+    "nOpt": 150,
+    "a0": 0.3,
+    "minPoolCost": 340000000,
+    "decentralisationParam": 1.0,
+    "maxBlockBodySize": 65536,
+    "maxBlockHeaderSize": 1100,
+    "maxTxSize": 16384,
+    "tau": 0.2,
+    "rho": 3.0e-3,
+    "poolDeposit": 500000000,
+    "keyDeposit": 2000000,
+    "minFeeB": 155381,
+    "minFeeA": 44,
+    "minUTxOValue": 1000000,
+    "extraEntropy": {
+        "tag": "NeutralNonce"
+    },
+    "eMax": 18
+}
+```
+
+The meaning of the fields is:
+
+| Field                 	| Initial Value                                                          	| Description                                                                                                                                                                                                                      	|
+|-----------------------	|------------------------------------------------------------------------	|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| protocolVersion       	| ```protocolVersion": {         "major": 2,         "minor": 2     }``` 	| Protocol version.  Minor versions indicate software updates (will generally be 0).  Major version 1 = Byron, 2 = Shelley                                                                                                         	|
+| nOpt                  	| 150                                                                    	| "Target number of pools" ("k").  Impacts saturation threshold, encouraging growth in number of stake pools.                                                                                                                                                                  	|
```

Comment:
now set to 500

Reply:
The auditors requested  the initial values so that we established a "chain of events".  The idea would be that future changes should refer to this as the basis for the change.  We should probably add a second CIP to record the current settings when we go live with community governance, since they have changed since the initial values

Reply:
I've added the changes since the hard fork